### DISCUSSION TITLE
Error when pushing changes to the QField Cloud

### DISCUSSION THREAD

**Discussion by SimonHHaas** [opengisch/QField#2781] [Q&A]
Hi,

I'm encountering a problem when trying to push changes to the QField Cloud, which only occurs under specific circumstances. 

I've got a parent .gpkg file with two child .csv tables. The .csv tables are used to store image-file paths and are linked to the .gpkg file.

When adding a new point to the .gpkg file without adding pictures to the .csv files in QField, pushing the changes to the QField Cloud works just fine. The status of the change is first listed as 'Started' on the QFieldCloud web interface ('Changes' section), after a short moment the status changes to 'Applied'. 

However, when adding points to the .gpkg file and also adding pictures to the child tables, synchronization stops working. The statuses of the changes to the .gpkg and .csv files are first listed as 'Started', however,  both change to 'Error' quickly. The image files themselves are being uploaded to the cloud successfully. 

I have tried setting the action of the failed changes to 'Apply', and clicked 'Save All'. Afterwards, I clicked 'Yes, apply pending changes', which lead to the error message 'Server Error (500)'. 

Here are a few screenshots of the failed changes in the 'Jobs' section, which might be useful:

(I don't know whether these screenshots contain any personal information, so I generously blurred them. I hope I didn't cover any important details)

![Screenshot4](https://user-images.githubusercontent.com/99903448/164090700-c164c40d-af40-4624-a5d4-310f5b8a90e6.png)

![Screenshot5](https://user-images.githubusercontent.com/99903448/164090715-94f64862-3240-434c-8aa0-77ed092f1c85.png)

![Screenshot6](https://user-images.githubusercontent.com/99903448/164090726-7b28c1db-183f-4184-b844-271e3f880b55.png)

Trying to do the same thing using the QGIS application on Windows 10 didn't work either. Under the 'Changes' section, the status also displays 'Error'. (However, in this case, there are no jobs with the type 'Delta Apply' in the 'Jobs' section. Instead, there are jobs with the type 'Process QGIS Project File', which all have the status 'Finished')

![Screenshot7](https://user-images.githubusercontent.com/99903448/164094888-a3e23678-3386-46b1-af1b-fe8b13bfdf6d.png)

I hope this description was detailed enough to help identify the problem. I'll gladly provide more information, in case it might be useful.

**Comment by Gstretch22** [comment]
Hi @SimonHHaas, 

I have recently had the same issue! did you manage to resolve this ?

**Comment by mia-snsh** [comment]
Hi, I also encountered the same issue. Did anyone have a solution for it?
